### DISCUSSION TITLE
New BUs not showing up to be retrieved

### DISCUSSION THREAD

**Discussion by MichalSkrisa** [Accenture/sfmc-devtools#1905] [Q&A] [answered]
- Two new Child BUs were created within our SFMC environment but didn't show up when I ran mcdev retrieve command
- I upgraded the mcdev few times as there were some updates lately but it didn't solve the problem
- Installed package has full access to both of those new BUs
- I'm backing up daily changes and want to add those two new BUs to the process

My findings:
Just out of curiosity, I made a new repo and ran the init mcdev command with the same credentials. The problem was solved and I see those new BUs to be retrieved. But what if I want to use my old repo already set up with bitbucket. Is there any solution to this?

**Comment by JoernBerkefeld** [comment, marked as answer]
hi Michal, sorry that i missed your question: when you add / remove BUs (or rename them and want to use the new names in mcdev) you then have to run `mcdev reloadBUs`. More details here: https://github.com/Accenture/sfmc-devtools/wiki/06.a-~-Admin-Commands#reloadbus

**Reply by MichalSkrisa** [reply]
Thank you @JoernBerkefeld for the insightful answer and I apologize for not having looked at the documentation properly at first :)
